peer_mentoring/peer_mentoring/settings.py

```diff
@@ -163,3 +163,12 @@
 REDIS_HOST = "localhost"
 REDIS_PORT = 6379
 REDIS_DB = 0
+
+# SENDGRID API
+
+SENDGRID_API_KEY = os.getenv("SENDGRID_API_KEY")
```

Comment:
might not need

Reply:
Deleted, that was based on the sendgrid django docs. not twilio.